generators/app/templates/config/locales/pt-BR/index.js

```diff
@@ -1,9 +0,0 @@
-import base from './base.yml';
-import models from './models.yml';
-
-let pt_BR = {
-  ...base['pt-BR'],
-  ...models['pt-BR']
-};
-
-export { pt_BR };
```

Comment:
I was thinking in https://translation.io/blog/gettext-is-better-than-rails-i18n, but anyway its easy to add this to a shared folder.